demos/react-dapp/src/App.tsx

```diff
@@ -589,8 +619,39 @@ const App: React.FC = () => {
             </button>
           </div>
         </section>
-        <hr />
-        <h2>Pairing and session management:</h2>
```

Comment:
I assume you want these removing as the button title for disconnecting the sessions is obvious enough?

Reply:
> I assume you want these removing as the button title for disconnecting the sessions is obvious enough?

haha no, I removed this by accident. Thanks for flagging